module/VuFind/src/VuFind/View/Helper/Root/AccountMenu.php

```diff
@@ -0,0 +1,377 @@
+<?php
+
+/**
+ * Account menu view helper
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Moravian library 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Josef Moravec <josef.moravec@mzk.cz>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\View\Helper\Root;
+
+use VuFind\ILS\Connection as IlsConnection;
+
+/**
+ * Account menu view helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Josef Moravec <josef.moravec@mzk.cz>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class AccountMenu extends \Laminas\View\Helper\AbstractHelper
+{
+    /**
+     * Constructor
+     *
+     * @param array $config AccountMenu configuration
+     */
+    public function __construct(protected array $config)
+    {
+    }
+
+    /**
+     * Get available menu items
+     *
+     * @return array
+     */
+    public function getItems(): array
+    {
+        return array_filter(
+            $this->config['MenuItems'] ?? $this->getDefaultItems(),
+            function ($item) {
+                return (isset($item['checkMethod']) && $this->{$item['checkMethod']}()) || !isset($item['checkMethod']);
```

Comment:
You could simplify this condition to:

`!isset($item['checkMethod']) || $this->{$item['checkMethod']}()`

If you check the unset condition first, then you can just assume the value is set after the `||`.

Reply:
done